host/integrationbase.go

```diff
@@ -141,9 +140,7 @@ func (s *IntegrationBase) setupSuite() {
 }
 
 func (s *IntegrationBase) setupLogger() {
-	zapLogger, err := zap.NewDevelopment()
-	s.Require().NoError(err)
-	s.Logger = loggerimpl.NewLogger(zapLogger)
+	s.Logger = loggerimpl.NewLoggerForTest(s.Suite)
```

Comment:
I _think_ I tried this a while ago and ran into lots of race conditions due to some background logging after test shutdown (e.g. gocql I think?).

So test it thoroughly for flakiness.  But if it's working, 👍 absolutely an improvement.

Reply:
I was trying to reduce the amount of logs tests produce here, but for tests we are always passing `-v` which turns on verbosity and all logs are emitted. I can leave it as it is, or revert the usage of zap test logger

Reply:
nah, if it works that's absolutely worth doing.  I just had problems doing similar things previously, but that was several months ago at least.